.circleci/config.yml

```diff
@@ -1209,6 +1217,34 @@ jobs:
             kurtosis engine status || true
 
             echo "Kurtosis setup complete"
+      # Start our devnet
+      - run:
+          name: Start devnet (<<parameters.devnet>>)
+          working_directory: kurtosis-devnet
+          command: |
+            echo "Starting <<parameters.devnet>> devnet..."
+            just <<parameters.devnet>>-devnet
+
+            # Check if the devnet started successfully
+            if ! kurtosis enclave ls | grep -q "<<parameters.devnet>>-devnet"; then
+              echo "Failed to start <<parameters.devnet>> devnet"
+              exit 1
+            fi
+
+            echo "<<parameters.devnet>> devnet started successfully"
+
+            # Wait a moment for the devnet to stabilize
+            sleep 5
+
+            # Print devnet info for debugging
+            kurtosis enclave inspect <<parameters.devnet>>-devnet || true
```

Comment:
Is the enclave name and the `just` command target always identical?

Reply:
For now at least, I believe so, yes.